tests/Unit/NumericalAlgorithms/Integration/Test_GslQuadAdaptive.cpp

```diff
@@ -0,0 +1,116 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
```

Comment:
This test isn't being built. You need to add `add_subdirectory(Integration)` to `tests/Unit/NumericalAlgorithms/CMakeLists.txt`